src/main/java/wolox/training/controller/BookController.java

```diff
@@ -95,11 +98,14 @@ public Book updateBook(@RequestBody Book book, @PathVariable Long id) {
         @RequestParam(required = false, defaultValue = "") String fromYear,
         @RequestParam(required = false, defaultValue = "") String toYear,
         @RequestParam(required = false, defaultValue = "") String pages,
-        @RequestParam(required = false, defaultValue = "") String isbn) {
+        @RequestParam(required = false, defaultValue = "") String isbn,
+        @RequestParam(required = false, defaultValue = "title") String sort,
+        @RequestParam(required = false, defaultValue = "0") Integer pageNumber,
+        @RequestParam(required = false, defaultValue = "5") Integer numberOfElements) {
+        Pageable customPageable = PageRequest.of(pageNumber, numberOfElements, Sort.by(sort));
```

Comment:
If you check the doc, you can simply add a Pageable to the arguments of the endpoint and the page, amount and sort stuff get's parsed automatically for you, so the only thing you do here at controller level, is take the pageable and send it to the repository ;)
Do it everywhere!

Reply:
Done!
Great, it's easier that way! Thanks!